osmcha/changeset.py

```diff
@@ -292,7 +293,10 @@ def autovandal(self):
         reason = 'Flagged by ML classifier'
 
         def load_model():
-            return joblib.load('models/autovandal.pkl')
+            directory = os.path.dirname(os.path.realpath(__file__))
+            filename = 'models/autovandal.pkl'
+            model = os.path.join(directory, filename)
```

Comment:
Fixed using absolute path string for loading the model by using python's `os` module. Thank you @batpad and @amishas157 for helping me debug the issue. Ex:

- directory: `/Users/bkowshik/repos/mapbox/osmcha/osmcha`
- filename: `models/autovandal.pkl`
- model: `/Users/bkowshik/repos/mapbox/osmcha/osmcha/models/autovandal.pkl`